pyproject.toml

```
@@ -64,17 +91,22 @@ include = ["dagfactory"]
 [tool.hatch.build.targets.wheel]
 packages = ["dagfactory"]
 
-[[tool.hatch.envs.tests.matrix]]
-python = ["3.9", "3.10", "3.11", "3.12"]
-airflow = ["2.8", "2.9", "2.10"]
+[tool.pytest.ini_options]
+filterwarnings = ["ignore::DeprecationWarning"]
+minversion = "6.0"
 
-[tool.hatch.envs.tests.scripts]
-static-check = " pre-commit run --files dagfactory/*"
+######################################
+# THIRD PARTY TOOLS
+######################################
 
 [tool.black]
 line-length = 120
 target-version = ['py39', 'py310', 'py311', 'py312']
 
+[tool.isort]
+profile = "black"
+known_third_party = ["airflow", "jinja2"]
```

I am not sure if we are using Jinja directly in this project
```suggestion
known_third_party = ["airflow"]
```